starlette_admin/contrib/odmantic/converters.py

```diff
@@ -109,6 +110,10 @@ def conv_bson_int64(self, *args: Any, **kwargs: Any) -> BaseField:
     def conv_bson_decimal(self, *args: Any, **kwargs: Any) -> BaseField:
         return DecimalField(**self._standard_type_common(**kwargs))
 
+    @converts("timedelta")
+    def conv_bson_timedelta(self, *args: Any, **kwargs: Any) -> BaseField:
+        return IntervalField(**self._standard_type_common(**kwargs))
+
```

Comment:
it seems odmantic does not support timedelta https://github.com/art049/odmantic/issues/120, we can remove this and just keep for mysql